examples/advanced_circuits_algorithms/Deutsch_Jozsa_Algorithm/Deutsch Jozsa Algorithm.ipynb

```diff
@@ -0,0 +1,1037 @@
+{
```

Comment:
<p>Once you've run the classical and quantum version of the random circuit, you can compare the results. If both correspond to the same type of function, then we have found another way to implement the constant or balanced function!  </p><p> </p><p>You can keep generating more random circuits to see how the constant and balanced functions can be created; including for circuits that are extremely large, for example 10 qubits!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/177/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>